src/sql/azdata.d.ts

```diff
@@ -1222,6 +1222,10 @@ declare module 'azdata' {
 		textIdentifier?: string | undefined;
 		encoding?: string | undefined;
 		formatted?: boolean | undefined;
+		freezeHeaderRow?: boolean | undefined;
```

Comment:
1. Please put this in azdata.proposed.d.ts. You'll create a duplicate interface definition there with ONLY the newly added props - they will be merged into a single definition at compile time
2. Please add doc comments to these (I realize that the other ones don't have them, but we're trying to enforce that new stuff at least has them and then going back and correcting older definitions as we go)